multigen/pipes.py

```diff
@@ -201,7 +206,7 @@ class Cond2ImPipe(BasePipe):
         "depth": "control_v11f1p_sd15_depth",
         "inpaint": "control_v11p_sd15_inpaint"
     }
-    cscalem = {
+    default_cs = {
```

Comment:
`default_cs` is a bad name. Why is "default" emphasized, but what "cs" is remains unclear?